### DISCUSSION TITLE
New Firmware v2.0.1 Not recognising BLTouch?

### DISCUSSION THREAD

**Discussion by bushybluejeans** [Jyers/Marlin#1589] [Q&A]
Hi, this is fantastic thanks!
I just flashed the new firmware to the Ender 3V2 and I notice that the BLTouch just blinks red and won't probe or move. I installed an older firmware version and that worked fine. Should I be compiling the new firmware myself?

Thank you!
Brendon

**Comment by rorywquin** [comment]
Try switching off and on again. Sometimes my probe does the same thing (however that was on 1.3.5) and I am just about to try 2.01

EDIT: Mine is working with 2.0.1

**Reply by bushybluejeans** [reply]
Thanks for checking that out and reporting back on it!

**Comment by Jyers** [comment]
Another good thing to try is resetting eeprom by selecting reset settings and then store settings is the control menu. Make sure to back up your settings first though

**Comment by bushybluejeans** [comment]
Thanks. I managed to get it working, then plugged the Raspberry pi in and it stopped working again. Not sure if that might be the issue or not? I also thought that maybe the pins aren't making contact so I've tried checking those by pulling them out individually but they seem secure. 

I tried the following:

1. Reset the EEPROM per Jyer's suggestion.
2. Turned it off and on again.
3. Reformatted the SD card and flashed the printer again, using a different filename. 

I'm beginning to think the  BLTouch is faulty at this stage.

**Comment by bushybluejeans** [comment]
Thanks, everybody. I ended up returning the BL Touch for now and reverting to V1.3.4 for now.

**Comment by bushybluejeans** [comment]
I agree. Firmware is firmware however in the case of the JyersUI, it's compiled by people who actually care a great deal about the quality of the product. That's why I've chosen it over other options out there.
